setup.py

```diff
@@ -12,8 +12,8 @@
 
 is_pypy = '__pypy__' in sys.builtin_module_names
 mpack_required = ['msgpack>=1.0.0']
-if is_pypy:
-    mpack_required.append('msgpack-pypy>=0.0.2')
+# if is_pypy:
+#    mpack_required.append('msgpack-pypy>=0.0.2')
```

Comment:
I see the project has been abandoned and there is no release since 2013, so I’m OK with removing this, but why just comment out?